src/nni_manager/training_service/remote_machine/remoteMachineTrainingService.ts

```diff
@@ -284,8 +301,14 @@ class RemoteMachineTrainingService implements TrainingService {
      */
     public async setClusterMetadata(key: string, value: string): Promise<void> {
         switch (key) {
+            case TrialConfigMetadataKey.NNI_MANAGER_IP:
+                this.nniManagerIpConfig = <NNIManagerIpConfig>JSON.parse(value);
+                break;
             case TrialConfigMetadataKey.MACHINE_LIST:
-                await this.setupConnections(value);
+                let localGpuMetricFolder: string = await this.generateGpuScript();
```

Comment:
naming convention. Rename 'localGpuMetricFolder' to an accurate, consistent and meaningful name.

Reply:
updated to gpuMetricCollectorScriptFolder